plugin/evm/validators_cache_test.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"github.com/flare-foundation/coreth/tests/mocks"
+	"github.com/flare-foundation/flare/ids"
```

Comment:
```suggestion
	"github.com/flare-foundation/coreth/tests/mocks"

	"github.com/flare-foundation/flare/ids"
```

I've seen in other PRs that people now split coreth and flare imports in two groups so I guess we should follow